code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -14,7 +14,7 @@ GLOBAL_LIST_EMPTY(PDAs)
 	righthand_file = 'icons/mob/inhands/misc/devices_righthand.dmi'
 	flags_1 = NOBLUDGEON_1
 	w_class = WEIGHT_CLASS_TINY
-	slot_flags = SLOT_ID | SLOT_BELT
+	slot_flags = SLOT_ID | SLOT_BELT | SLOT_PDA

```

Comment:
Do we want to remove the `SLOT_ID` from this as well. Pretty sure its that way elsewhere? (I could be entirely wrong)

Reply:
This is how it is on Paradise.